.github/workflows/build-sample-apps.yml

```diff
@@ -0,0 +1,234 @@
+name: Build sample apps
+
+on:
+  pull_request: # build sample apps for every commit pushed to an open pull request (including drafts)
+  push:
+    branches: [ main, feature/* ]
```

Comment:
Does this mean that the workflow will trigger on any branch that matches this pattern `feature/*` ? 

If yes, then what about other branches that do not follow this pattern ?

Reply:
Yes. The workflow will trigger only on Pull Requests, or any commits pushed. to `main` or branches with pattern `feature/*`, similar to workflows in our other repose ([iOS](https://github.com/customerio/customerio-ios/blob/b7d7d4fec2bd4be9f25b3b8937b01c7dc5360e0a/.github/workflows/build-sample-apps.yml#L4-L6), [Android](https://github.com/customerio/customerio-android/blob/a68a9c5c872dad3b9ea4ba951806a82b8268ad2b/.github/workflows/build-sample-apps.yml#L4-L6) and [Flutter](https://github.com/customerio/customerio-flutter/blob/97d92e76af0cd1486b9444be57f3766322c46fb6/.github/workflows/build-sample-apps.yml#L4-L6)) today.